File: comms/push-notifications/credentials/fetch-credential/fetch-credential.go
package example

import (
    context "context"

    client "github.com/twilio/twilio-comms-go/twilio/client"
    option "github.com/twilio/twilio-comms-go/twilio/option"
)

func do() {
    client := client.NewWithOptions(
        option.WithBasicAuth(
            "<username>",
            "<password>",
        ),
    )
    client.PushNotifications.Credentials.Fetch(
        context.TODO(),
        "comms_credential_01h9krwprkeee8fzqspvwy6nq8",
    )
}

File: comms/push-notifications/credentials/fetch-credential/fetch-credential.java
package com.example.usage;

import com.twilio.sdk.TwilioComms;

public class Example {
    public static void main(String[] args) {
        TwilioComms client = TwilioComms
            .builder()
            .credentials("TWILIO_ACCOUNT_SID", "TWILIO_AUTH_TOKEN")
            .build();

        client.pushNotifications().credentials().fetch("comms_credential_01h9krwprkeee8fzqspvwy6nq8");
    }
}